examples/widgets/tabview/lv_example_tabview_2.py

```diff
@@ -0,0 +1,48 @@
+def scroll_begin_event(e):
+
+    #Disable the scroll animations. Triggered when a tab button is clicked */
+    if e.get_code() == lv.EVENT.SCROLL_BEGIN:
+        a = lv.anim_t.cast(e.get_param())
```

Comment:
This needs to be changed to `__cast__` (https://github.com/lvgl/lv_binding_micropython/issues/157#issuecomment-870389807).

Reply:
In fact, this is what I had in the first place, but I get:
![image](https://user-images.githubusercontent.com/1673533/124358024-a9c01700-dc1e-11eb-8987-b26c921e5fdf.png)

Reply:
Your MicroPython version is too old (June 27); the change was made on June 29.

Reply:
Upgraded my MicroPython version and modified example as you said.